force-app/main/default/classes/AccountNamingSettingsService_TEST.cls

```diff
@@ -0,0 +1,217 @@
+    @isTest 
+    private static void getHouseholdAccountNamingFormatModelValid() {
+        AccountNamingSettingsMapper.instance = new STUB_AccountNamingSettingsMapper();
+        AccountNamingSettingsService accountNamingSettingsServiceInstance = AccountNamingSettingsService.getInstance();
+        
+        Test.startTest();
+        AccountNamingFormatModel accountNamingFormatModel = accountNamingSettingsServiceInstance.getHouseholdAccountNamingFormatModel();
+        Test.stopTest();
+
+        System.assertEquals(
+            'hh format 1', 
+            accountNamingFormatModel.validAccountNamingFormatsByName.get('hhFormat1'),
+            'Value for \'hhFormat1\' hh account format should match mapped value.'
+        );
+
+        System.assertEquals(
+            'hh format 2', 
+            accountNamingFormatModel.validAccountNamingFormatsByName.get('hhFormat2'),
+            'Value for \'hhFormat2\' hh account format should match mapped value.'
+        );
+    }
+
+    /**************************************************************************************************************************
+    ****************************************************** STUBS **************************************************************
+    **************************************************************************************************************************/
+
+    /**************************************************************************************************************************************
+    * @description Stub class to simulate the response from AccountNamingSettingsMapper to return a valid AdministrativeAccountSettingsModel.
+    **************************************************************************************************************************************/ 
+    private class STUB_AccountNamingSettingsMapper extends AccountNamingSettingsMapper { 
+        public override AccountNamingSettingsModel getAccountNamingSettingsModel() {
+            return new AccountNamingSettingsModel(
+                'administrativeAccountNameFormat',
+                'householdAccountNamingFormat',
+                'customAdministrativeAccountNameFormat',
+                'customHouseholdAccountNamingFormat',
+                false
+            );
+        }
+
+        public override AccountNamingFormatModel getAdministrativeAccountNamingFormatModel() {
+            return new AccountNamingFormatModel(
+                new Map<String, String>{
+                    'adminFormat1' => 'admin format 1',
+                    'adminFormat2' => 'admin format 2'
+                }
+            );
+        }
+
+        public override AccountNamingFormatModel getHouseholdAccountNamingFormatModel() {
+            return new AccountNamingFormatModel(
+                new Map<String, String>{
+                    'hhFormat1' => 'hh format 1',
+                    'hhFormat2' => 'hh format 2'
+                }
+            );
+        }
+    }
+}
+
+
```

Comment:
This should be resolved if you run a retrieve changes. In this instance use dx_push to push up your changes after deploying from feature/232